File: MODUL02/PraktikumPy/PRAK204-2310817110004-AbdurrahmanGilangHarjuna.py
r = float(input("Masukkan jari-jari bejana: "))
h = float(input("Masukkan tinggi bejana: "))
pi = 22/7
volume = pi * r**2 * h
luas = 2 * pi * r * (r + h)
keliling = 2 * pi * r

volume = round(volume, 2)
luas = round(luas, 2)
keliling = round(keliling, 2)

print("Volume = {:.2f}".format(volume))
print("Luas = {:.2f}".format(luas))
print("Keliling = {:.2f}".format(keliling))

File: MODUL02/PraktikumC/PRAK204-2310817110004-AbdurrahmanGilangHarjuna.c
#include <stdio.h>
#include <math.h>

int main() {
    double r, h;
    printf("Masukkan jari-jari bejana: ");
    scanf("%lf", &r);
    printf("Masukkan tinggi bejana: ");
    scanf("%lf", &h);

    double volume, luas, keliling;
    const double pi = 22/7;
    volume = r * r * h * pi;
    luas = 2 * r * (r + h) * pi;
    keliling = 2 * r * pi;
    
    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf\n", keliling);

    return 0;
}
